src/sciwyrm/main.py

```diff
@@ -19,13 +20,28 @@
     notebook_template_path,
 )
 
+import logging
+
 app = FastAPI()
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler())
+
+
+@app.get("/alive", response_description="The service is alive")
+async def list_templates(
+    config: Annotated[AppConfig, Depends(app_config)]
+) -> bool:
+    """Return a list of available notebook templates."""
+    return True
```

Comment:
```suggestion
@app.get("/alive", response_description="The service is alive")
async def is_alive() -> bool:
    """Return True to indicate that the service is running."""
    return True
```

Reply:
How  about using the more conventional name live(z) and returning the string ok

Reply:
I renamed it to `livez`.